server.js

```diff
@@ -2618,6 +2618,52 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// dotnet-status integration.
+camp.route(/^\/dotnetstatus\/(.+)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var projectUri = match[1]; // gh/{USER}/{REPO}/{PROJECT}
+  var format = match[2];
+  var url = 'http://dotnet-status.com/api/status/' + projectUri + '/';
+  var badgeData = getBadgeData('dependencies', data);
+  request(url, function (err, res, buffer) {
+    if (err != null) {
+      badgeData.text[1] = 'inaccessible';
+      sendBadge(format, badgeData);
+      return;
+    }
+    if (res.statusCode === 404) {
+      badgeData.text[1] = 'not found';
```

Comment:
Under what circumstances would this error be raised?

Reply:
If the project could not be located (possibly caused by a typo)

Reply:
Why is there a separate case for `'project not found'`?

Reply:
ah, sorry for the confusion I answered on mobile w/o taking a closer look. the API application will not issue a 404, but the webserver will if the request cannot be routed.

Reply:
That sounds perfect!